docs/getting_started.md

```diff
@@ -281,6 +281,7 @@ components.
 | ACTS_BUILD_PLUGIN_FPEMON            | Build FPE monitoring plugin<br> type: `bool`, default: `OFF`                                                                                                                                                                       |
 | ACTS_BUILD_PLUGIN_GEOMODEL          | Build GeoModel plugin<br> type: `bool`, default: `OFF`                                                                                                                                                                             |
 | ACTS_BUILD_PLUGIN_TRACCC            | Build Traccc plugin<br> type: `bool`, default: `OFF`                                                                                                                                                                               |
+| ACTS_TRACCC_ENABLE_CUDA             | Enable CUDA for the traccc plugin<br> type: `bool`, default: `OFF`                                                                                                                                                                 |
```

Comment:
```suggestion
| ACTS_PLUGIN_TRACCC_ENABLE_CUDA             | Enable CUDA for the traccc plugin<br> type: `bool`, default: `OFF`                                                                                                                                                                 |
```
?

Reply:
I'm open to renaming it; right now it is named in line with the EXATRKX options, e.g. `ACTS_EXATRKX_ENABLE_ONNX`. What do you prefer?

Reply:
If there's precedent, let's keep it like this.